cmd/app/options/app.go

```diff
@@ -39,6 +43,12 @@ func (k *KubeOIDCProxyOptions) AddFlags(fs *pflag.FlagSet) *KubeOIDCProxyOptions
 	fs.IntVarP(&k.ReadinessProbePort, "readiness-probe-port", "P", 8080,
 		"Port to expose readiness probe.")
 
+	fs.DurationVar(&k.FlushInterval, "flush-interval", time.Millisecond*50,
+		"Specifies the interval to flush request bodies. If 0ms, "+
+			"no periodic flushing is done. A negative value means to flush "+
+			"immediately after each write. Ignored, and flushed immediately, when "+
```

Comment:
Maybe rephrase as "Streaming requests such as 'kubectl exec' will ignore this option."? Took me a few reads over to get it :)